src/scenes/home/opCodeCon/opCodeCon.css

```diff
@@ -0,0 +1,12 @@
+.hero {
+  background-image: url('./raleigh.jpg');
+  text-align: left;
+}
+.heading {
+  color: #47566B;
+  margin: 10%;
+}
+.textLink {
+  color: #47566B;
+  font-size: 1.2em;
+}
```

Comment:
Are there any colors in our palette that would be good hover states for the the .textLink class?

Reply:
Colors are kind of limited to have them stand out, it's either the red (which I attempted) or the grey/ blue ones.